src/briefcase/integrations/android_sdk.py

```diff
@@ -1003,7 +1004,7 @@ def start_emulator(self, avd):
                 "8.8.8.8",
             ],
             env=self.env,
-            stdout=subprocess.PIPE,
+            stdout=subprocess.DEVNULL,
             stderr=subprocess.PIPE,
             start_new_session=True,
```

Comment:
Sending data to `PIPE` without intending to read it is probably bad practice; most notably since a process can deadlock if the process' buffer fills. While that's unlikely here, removing the possibility by sending it to `DEVNULL` seems like nice insurance.